spec/services/teachers/induction_status_spec.rb

```diff
@@ -1,7 +1,5 @@
-require "rails_helper"
-
 RSpec.describe Teachers::InductionStatus do
-  subject { Teachers::InductionStatus.new(teacher:, induction_periods:, trs_induction_status:) }
+  subject(:service) { described_class.new(teacher:, induction_periods:, trs_induction_status:) }
```

Comment:
Not really a problem but I've seen newer devs get confused by `described_class` and save it for shared contexts - but we're not really being consistent either way yet!